crates/mcp/src/surfpool/start_surfnet.rs

```diff
@@ -10,10 +12,28 @@ pub struct StartSurfnetResponse {
 
 #[derive(Serialize)]
 pub struct StartSurfnetSuccess {
-    pub surfnet_url: String,
+    pub kind: String,
```

Comment:
Instead of an string with hard-coded values, can we use an enum with `Command` and `SubProcess` options?

Reply:
Also, since the `command` is an optional that is only used when the `kind` is "command", your enum should contain that data:
```rust
enum StartSurfnetSuccessKind {
    Command(SerializedCommand),
    SubProcess
}
```